### PR TITLE
pfd: add a line for "real attitude" in PFD

### PR DESCRIPTION
**Pull request by mlyle**
This is just janitorial work, fixing up older code I've been carrying out-of-tree.

### CODE REVIEW

**Comment by mlyle** [comment]
jenkins, build artifacts please

**Comment by mlyle** [comment]
jenkins, build artifacts please

File: ground/gcs/src/plugins/uavobjects/uavdataobject.h [hunk 1]
```diff
@@ -37,7 +37,7 @@
 class UAVOBJECTS_EXPORT UAVDataObject : public UAVObject
 {
     Q_OBJECT
-
+    Q_PROPERTY(bool isPresent READ getIsPresentOnHardware WRITE setIsPresentOnHardware NOTIFY presentOnHardwareChanged)
```
Comment (tracernz) [inline ground/gcs/src/plugins/uavobjects/uavdataobject.h]: We should have exactly the same name as the c++ code in all our QML stuff or things will get messy. (i.e. `isPresentOnHardware`.

File: ground/gcs/share/pfd/default/Pfd.qml [hunk 2]
```diff
@@ -53,7 +53,13 @@ Rectangle {
             Loader {
                 id: worldLoader
                 anchors.fill: parent
-                source: qmlWidget.terrainEnabled ? "PfdTerrainView.qml" : "PfdWorldView.qml"
+                source: "PfdWorldView.qml"
+            }
+
+            Loader {
+                id: simAttiLoader
```
Comment (tracernz) [inline ground/gcs/share/pfd/default/Pfd.qml]: `simAttitudeLoader` but no refs to it anyway so just delete?

File: ground/gcs/share/pfd/default/PfdAttitudeLine.qml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+        gradient: Gradient {
+            GradientStop { position: 0.496;    color: "#00000000" }
+            GradientStop { position: 0.500;    color: "#c0008000" }
+            GradientStop { position: 0.504;    color: "#00000000" }
+        }
+
+        transform: [
+            Translate {
+                id: pitchTranslateReallAtt
```
Comment (tracernz) [inline ground/gcs/share/pfd/default/PfdAttitudeLine.qml]: Typo etc. But this doesn't actually need an id/I can't see any references to it?

**Comment by mlyle** [comment]
Thx for review-- fixed up.  (Shouldn't need a new artifact build).

**Review by tracernz — APPROVED** [review]
Thanks. :+1:

**Pull request merged**